cmd/main.go

```diff
@@ -41,6 +41,7 @@ func main() {
 		volMetricsFsRateLimit    = flag.Int("vol-metrics-fs-rate-limit", 5, "Volume metrics routines rate limiter per file system")
 		deleteAccessPointRootDir = flag.Bool("delete-access-point-root-dir", false,
 			"Opt in to delete access point root directory by DeleteVolume. By default, DeleteVolume will delete the access point behind Persistent Volume and deleting access point will not delete the access point root directory or its contents.")
+		tags = flag.String("tags", "", "Space separated key:value pairs which will be added as tags for EFS resources. For example, 'environment:prod region:us-east-1'")
```

Comment:
this looks like it needs gofmt.

I am not sure I like the accepted format,  but I don't have a better one to suggest. I checked the awscli convention and it's more verbose and harder to parse...:
`Key=string,Value=string ...`

Reply:
This is how gofmt like it. I tried padding with enough spaces to match above variables, but `make verify` failed.

I feel `key:value` is easier to parse than `key=string,value=string` for multiple tags